support.org

```diff
@@ -1575,26 +1574,32 @@ Use [[https://github.com/seagle0128/doom-modeline][doom-modeline]] to ornament t
 #+end_src
 
 ** api
-*** remote-exists?
+*** remote-exists-p
```

Comment:
Why are you making these internal?

And if there's a good reason for it, you should move them to an `internal` heading sibling to the `api` heading, just like in the `core` section.

They seem like they would be useful functions for people customizing Episteme locally.